src/libs/layout/unlock-vault/index.tsx

```diff
@@ -150,6 +153,32 @@ export function UnlockVaultPageContent() {
     );
   }
 
+  if (isLoading) {
+    return (
+      <ContentContainer>
+        <CenteredFlexColumn style={{}}>
+          <Player
+            speed={1}
+            autoplay
+            keepLastFrame
```

Comment:
What do you think if we add `loop` props?
If the unlocking process is long, it will animate all time. Now it only animates once and stops.
Or this will decrease unlocking performance?

Reply:
It'll not have a performance impact, but It depends on the animation itself if it will look good when looped, I'm still waiting for the final animation but it's a valid idea :)

Reply:
I set the unlocking timeout for 10 sec and it looks weird when the animation stops and nothing changes

Reply:
It shouldn't take more than 3 sec even on older laptops. There will be a low percentage of pretty obsolete machines but there is nothing we can do about that.